.vscode/launch.json

```diff
@@ -10,11 +10,7 @@
       "request": "launch",
       "module": "homeassistant",
       "justMyCode": false,
-      "args": [
-        "--debug",
-        "-c",
-        "config"
-      ],
+      "args": ["--debug", "-c", "config"],
```

Comment:
Unrelated change?

Reply:
was done automatically upon save by vscode, will undo

Reply:
the instant I hit save it automatically reformats it this way, does it do the same to anyone else? 😅
could it be something configured with prettier extension shipped with the repo?

Reply:
since this was done automatically by the ide because I had added a task to run and debug alone the failing task but that will be moved to a different PR, i've reverted the whole from dev branch

Reply:
found out root cause, it's vscoce prettier plugin that decides how to format arrays based on their lenght...and as absurd it may seem there is no way to turn that behavior off so I had to disable the whole plugin itself 🤦🏻‍♂️
https://github.com/prettier/prettier-vscode/issues/352